web/src/components/MalteHelpForm.tsx

```diff
@@ -209,17 +227,18 @@ const MalteHelpForm = ({ submit, dashboardRoute }: MalteHelpFormProps): ReactEle
           />
         </RadioGroup>
         <Divider />
-        <InputSection
-          id='comment'
-          title={t('causeOfContact')}
-          subtitle={t('maxCharacters', { numberOfCharacters: 200 })}
-          multiline
-          value={comment}
-          onChange={setComment}
-          optional
-          maxLength={200}
-          submitted={submitted}
-        />
+        <InputSection id='comment' title={t('contactReason')}>
+          <Input
+            id='comment'
+            hint={t('maxCharacters', { numberOfCharacters: 200 })}
```

Comment:
I think moving this to a shared constant in the api-client would be nice. Perhaps just in the same file as the `submit` happens.